docs/resources/custom_location.md

```diff
@@ -0,0 +1,46 @@
+---
+page_title: "cpln_custom_location Resource - terraform-provider-cpln"
+subcategory: "Custom Location"
+description: |-
+---
+
+# cpln_custom_location (Resource)
+
+Manages an org's [Custom Location](https://docs.controlplane.com/reference/location#byok-locations).
+
+## Declaration
+
+### Required
+
+- **name** (String) Name of the Custom Location.
+- **enabled** (Boolean) Indication if custom location is enabled.
+- **provider** (String) Provider of the custom location, must be `byok`.
```

Comment:
This can't be `provider`, it should match what we have defined in the resource. Please change it to `cloud_provider`.